libknet/logging.c

```diff
@@ -20,7 +20,7 @@
 #include "logging.h"
 #include "threads_common.h"
 
-static struct pretty_names subsystem_names[] =
+static struct pretty_names subsystem_names[KNET_MAX_SUBSYSTEMS] =
```

Comment:
Are you sure this shouldn't be + 1 (as with KNET_LOG_DEBUG)?

Reply:
MAX_SUBSYSTEMS has already +1 included.